dev_guide/builds.adoc

```diff
@@ -366,13 +370,14 @@ trigger on:
 ====
 
 ----
-{
-  "metadata":{
-    "name": "ruby-20-centos7",
-  },
   "kind": "ImageStream",
-  "apiVersion": "v1beta1",
-}
+  "apiVersion": "v1",
+  "metadata": {
+    "name": "ruby-20-centos7"
+  },
+  "status": {
+    "dockerImageRepository": ""
+  }
```

Comment:
ooops, nevermind. it's going to use the default internal registry and the metadata.name.